OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -1670,3 +1677,20 @@ void testExternalForce()
             ASSERT_EQUAL(def, val, 10*accuracy);
     }
 }
+
+void testSerializeDeserialize() {
```

Comment:
> The test just makes sure that property values are same regardless of which version the model file is.

I don't think it does. Suppose a model has 5 "enabled" and 5 "disabled" muscles. The test will still pass if the `updateFromXMLNode()` code always erroneously set the `appliesForce` property to its default value of `true`.